nBTC/sources/nbtc.move

```diff
@@ -20,16 +20,18 @@ const DECIMALS: u8 = 8;
 const SYMBOL: vector<u8> = b"nBTC";
 const NAME: vector<u8> = b"Native BTC";
 const DESCRIPTION: vector<u8> = b"Natvie synthetic BTC";
-const ICON_URL: vector<u8> = b"https://raw.githubusercontent.com/gonative-cc/sui-native/master/assets/nbtc.svg";
+const ICON_URL: vector<u8> =
+    b"https://raw.githubusercontent.com/gonative-cc/sui-native/master/assets/nbtc.svg";
 
 // Configuration
 /// The Object ID of the Bitcoin SPV Light Client.
-const LIGHT_CLIENT_ID: address = @0xCA;
+const LIGHT_CLIENT_ID: address = @light_client;
 /// The fallback Sui address to receive nBTC if OP_RETURN data is invalid or missing.
-const FALLBACK_ADDR: address = @0xCF;
+const FALLBACK_ADDR: address = @fallback;
```

Comment:
Fallback can't empty. We can set the fallback as the address we want to use for this nBTC.